client/client-multi/client-c/src/kaa/platform/kaa_client_properties.h

```diff
@@ -24,8 +24,28 @@
 #ifndef POSIX_KAA_CLIENT_PROPERIES_H_
 #define POSIX_KAA_CLIENT_PROPERIES_H_
 
+#include <kaa_error.h>
+
 typedef struct {
-        unsigned long max_update_time;
+    const char *working_directory;
 } kaa_client_props_t;
 
+/**
+ * @brief      sets the client kaa properties struct by the user-defined struct
+ *
+ * @param[in]  self  pointer to the user-defined structure
+ * 
+ * @return     error code
+ */
+kaa_error_t kaa_client_props_set(const kaa_client_props_t *self);
+
+/**
+ * @brief      gets current storage folder path
+ * 
+ * @note       path must end with "/" (slash)
```

Comment:
The user can not control this in the call to the function. Move this doc to `kaa_client_props_set()` or `kaa_client_props_t`.